Makefile

```diff
@@ -1,8 +1,8 @@
 SHELL=/bin/bash -euo pipefail
 
-install: install-node install-python install-hooks
+install: install-node install-poetry install-hooks
 
-install-python:
+install-poetry:
```

Comment:
Is anything dependent on this make target? I'd guess most things would use `install` but might be worth double checking.

Reply:
This was just a name change, because this target was installing python dependencies and not python per se. There's nothing dependent on it